src/spaInterface/spaOutputsForRunIds.ts

```diff
@@ -31,7 +31,7 @@ export const updateSpaOutputForRun = async (runId: string) => {
     const resp = await postStanPlaygroundRequest(req)
     if (resp.type !== 'getProjectFile') {
         console.warn(resp)
-        throw Error('Unexpected response from Stan Playground')
+        throw Error('Unexpected response from Stan Playground while retrieving project file')
```

Comment:
I made the error messages more specific to make it easier to distinguish the different test cases.